dotnet-base-libraries/ReturnInsertedIdentityWithDapper/ReturnInsertedIdentityWithDapper/Methods/Methods.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Dapper;
+using Microsoft.Data.SqlClient;
+
+namespace ReturnInsertedIdentityWithDapper;
+
+public class Methods
+{
+    public static int UseOfOutputInserted(string connectionString, Student newStudent)
+    {
+        using var dbConnection = new SqlConnection(connectionString);
+
+        dbConnection.Open();
+
+        var lastInsertedId = dbConnection.QuerySingle<int>(
+            $"INSERT INTO Students (Firstname, Surname) " +
+            $"OUTPUT INSERTED.Id " +
+            $"VALUES ('{newStudent.Firstname}', '{newStudent.Surname}'); ",
```

Comment:
Instead of concatenating multiline strings with `+` signs, can we use a more sophisticated approach like [raw string literals](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/raw-string)?

Reply:
Used string literal.